internal/web/routes/routes.go

```diff
@@ -0,0 +1,18 @@
+package routes
```

Comment:
Consider adding error handling for the Echo routes, especially for asynchronous operations. Currently, there is no indication of how potential errors from the GitHub API or OpenAI API calls are being handled. This could lead to unhandled exceptions or degraded user experience.